queries.md

```diff
@@ -2,22 +2,67 @@
 
 ## find all customers that live in London. Returns 6 records.
 
+select * from customers
+where city = 'London';
+
 ## find all customers with postal code 1010. Returns 3 customers.
 
+select * from customers
+where PostalCode = 1010;
+
+select * from customers
+where PostalCode = '1010';
+
 ## find the phone number for the supplier with the id 11. Should be (010) 9984510.
 
+select Phone from Suppliers
+where SupplierID = 11;
+
+select * from Suppliers
+where SupplierID = 11;
+
 ## list orders descending by the order date. The order with date 1997-02-12 should be at the top.
 
+select * from Orders
+order by OrderDate desc;
+
 ## find all suppliers who have names longer than 20 characters. You can use `length(SupplierName)` to get the length of the name. Returns 11 records.
 
+select * from Suppliers
+where length(supplierName) > 20;
+
+
 ## find all customers that include the word "market" in the name. Should return 4 records.
 
+select * from Customers
+where CustomerName like '%market%';
+
 ## add a customer record for _"The Shire"_, the contact name is _"Bilbo Baggins"_ the address is _"1 Hobbit-Hole"_ in _"Bag End"_, postal code _"111"_ and the country is _"Middle Earth"_.
 
+insert into Customers (CustomerName, ContactName, Address, City, PostalCode, Country)
+values ('The Shire','Bilbo Baggins', '1 Hobbit-Hole', 'Bag End', '111', 'Middle Earth');
+
 ## update _Bilbo Baggins_ record so that the postal code changes to _"11122"_.
 
+update customers
+set PostalCode = '11122'
+where ContactName ='Bilbo Baggins';
+
 ## list orders grouped by customer showing the number of orders per customer. _Rattlesnake Canyon Grocery_ should have 7 orders.
 
+select orders.CustomerID, Customers.CustomerName
+
+
+group by CustomerName
+order by count(CustomerID) desc;
+
+select orders.CustomerID, customers.CustomerName, count(customers.customerID)
+from Orders
+inner join Customers
+on Orders.CustomerID=Customers.CustomerID;
```

Comment:
Using aliases here would make this a good bit easier/shorter. This also looks a little off, but close!